www/docs/usage/config-files.md

```diff
@@ -53,28 +56,27 @@ bin = "$HOME/bin/terraform"
 version = "0.11.3"
 ```
 
-6. Run `tfswitch` and it should automatically install the required terraform version in the specified binary path
+6. Run `tfswitch` and it should automatically install the required version in the specified binary path
 
-**NOTE**
-
-1. For Linux users that do not have write permission to `/usr/local/bin/`, `tfswitch` will attempt to install terraform at `$HOME/bin`. Run `export PATH=$PATH:$HOME/bin` to append bin to PATH
-2. For Windows host, `tfswitch` need to be run under `Administrator` mode, and `$HOME/.tfswitch.toml` with `bin` must be defined (with a valid path) as minimum, below is an example for `$HOME/.tfswitch.toml` on windows
+Below is an example for `$HOME/.tfswitch.toml` on Windows:
 
 ```toml
 bin = "C:\\Users\\<%USRNAME%>\\bin\\terraform.exe"
 ```
 
-## Setting the default version using `.tfswitch.toml` file
+## Setting the default (fallback) version using `.tfswitch.toml` file
 
-The `.tfswitch.toml` file can be configured with a `default-version` attribute to configure tfswitch a particular version, if no other sources of versions are found
+If `tfswsitch` is unable to determine the version to use, it errors out.  
```

Comment:
```suggestion
By default, If `tfswsitch` is unable to determine the version to use, it errors out.  
```
Taken out of context, this probably isn't great and is the first thing someone will read.

Reply:
Thanks 👍🏻 
b11152a